boards/hardkernel/odroid_go/odroid_go_procpu.dts

```diff
@@ -155,6 +155,7 @@
 			disk-name = "SD";
 			status = "okay";
 		};
+
```

Comment:
>Formatting: Further refinement to dts-linter

Use dts: for treewide devicetree changes, https://docs.zephyrproject.org/latest/contribute/guidelines.html#pull-request-guidelines

Reply:
I guess this comment was intend for this PR that has now been merged https://github.com/zephyrproject-rtos/zephyr/pull/100247/commits. Noted will be more descriptive in my commit titles